Formula/lib/libmsquic.rb

```diff
@@ -0,0 +1,45 @@
+class Libmsquic < Formula
+  desc "Cross-platform, C implementation of the IETF QUIC protocol"
+  homepage "https://github.com/microsoft/msquic"
+  url "https://github.com/microsoft/msquic.git",
+  tag:      "v2.4.4",
+  revision: "e87ae9a37bcebdf5ab3725e5d882ddb50e4dbbb4"
```

Comment:
```suggestion
  url "https://github.com/microsoft/msquic.git",
      tag:      "v2.4.4",
      revision: "e87ae9a37bcebdf5ab3725e5d882ddb50e4dbbb4"
```
Do we need a Git clone? Can we use a tarball?